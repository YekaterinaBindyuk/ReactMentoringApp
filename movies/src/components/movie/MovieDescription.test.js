import React from "react";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const description  = 'description';
  const tree = renderer
    .create(<div className="description">{description}</div>)
    .toJSON();
  expect(tree).toMatchSnapshot(description);
});
