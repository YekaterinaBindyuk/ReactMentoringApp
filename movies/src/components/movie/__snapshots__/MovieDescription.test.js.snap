// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: description 1`] = `
<div
  className="description"
>
  description
</div>
`;
